scenarios/04_Battle_of_Barnon.cfg

```diff
@@ -1224,6 +1274,93 @@
+        [if]
+            [variable]
+                name=ano_barnon_tr
+                equals=no
+            [/variable]
+            [and]
+                [have_unit]
+                    id=Bor Cryne
+                    [filter_side]
+                        [allied_with]
+                            side=1
+                        [/allied_with]
+                    [/filter_side]
+                [/have_unit]
+            [/and]
+            [and]
+                [have_unit]
+                    id=Uri van Roe
+                    [filter_side]
+                        [allied_with]
+                            side=1
+                        [/allied_with]
+                    [/filter_side]
+                [/have_unit]
+            [/and]
+            [then]
+                {MSG_Uri _"...well, now that Gawen thinks he's won, there's no need to keep up this farce, is there, Bor?"}
+                [modify_side]
+                    side=4
+                    team_name=bad
+                [/modify_side]
+
+                {REPLACE_SCENARIO_MUSIC theme_of_a_new_order.ogg}
+                {MSG_Cryne _"Indeed, Uri, there isn't! Let our swords instead cut that underling's head off! And slay that witch, his step-mother, as well!"}
+                [modify_side]
+                    side=3
+                    team_name=bad
+                [/modify_side]
+                {APPEND_MUSIC battle/onethousandsuns.ogg}
+                {APPEND_MUSIC battle/battlecry.ogg}
+                {APPEND_MUSIC battle/gameplay06.ogg}
+                {APPEND_MUSIC siege_of_laurelmor.ogg}
+                {APPEND_MUSIC battle-epic.ogg}
+                # This macro (ANO4_TRAITORS) is defined in ano-01_06macros.cfg and contains most of the dialogue that occurs here:
+                {ANO4_TRAITORS _"Reach either the eastern or western signpost with both Lorin AND Reme. After that, hold on at least $ano_howmanyturns turns before the honorable death of Gawen Hagarthen."}
+            [/then]
+            [else]
+#ifdef DEBUG_MODE
+                {IF_DEBUG_MODE_IS_ACTUALLY_ON}
+                {MSG_narrator _"Using debug mode to kill all the enemies is cheating!"}
+                {ELSE}
+                {MSG_narrator _"DEBUG_MODE is defined; that means you probably cheated to kill all the enemies!"}
+                {END_IF}
+#else
+                {MSG_narrator _"Despite the incredible odds stacked against him, Gawen somehow managed to defeat all of his enemies at Barnon."}
+                {MSG_narrator _"However, this meant that he never managed to unite the warring factions of Wesnoth against the common foe that was due to invade..."}
+#endif
+                {MSG_narrator _"This is an early end to the campaign. If you would like to see the rest of it, please replay this scenario and let Gawen get defeated as he is supposed to be."}
```

Comment:
Think it should be "be defeated".

Reply:
changed in [869ac7b](https://github.com/nemaara/A_New_Order/pull/76/commits/869ac7b2a6b29fd86c6006372db3ea82455dd608)